### PR TITLE
[FD 381] apps mobile layout /1

### PR DESCRIPTION
**Pull request by starkyru**

### CODE REVIEW

File: src/components/PanelWithHeader/PanelWithHeader.tsx [hunk 1]
```diff
@@ -41,7 +49,8 @@ const styles = (theme: Theme): StyleRules => ({
       padding: 0,
     },
     [theme.breakpoints.down('xs')]: {
-      padding: `${theme.spacing.unit * 2}px 0`,
+      // padding: `${theme.spacing.unit * 2}px 0`,
```
Comment (BigBadAlien) [inline src/components/PanelWithHeader/PanelWithHeader.tsx]: //

File: src/components/PanelWithHeader/PanelWithHeader.tsx [hunk 2]
```diff
@@ -116,6 +134,10 @@ const PanelWithHeaderComponent: React.FunctionComponent<
   );
 };
 
+PanelWithHeaderComponent.defaultProps = {
+  variant: 'panel',
```
Comment (BigBadAlien) [inline src/components/PanelWithHeader/PanelWithHeader.tsx]: What's a sense to use `defaultProps` syntax?

File: src/components/SwapTokens/DepositAddress/DepositAddress.tsx [hunk 3]
```diff
@@ -229,7 +229,7 @@ const DepositAddressComponent: React.FunctionComponent<
         />
       </div>
     ) : (
-      <Error message={t('save-credentials.unexpected-error')} center={true} />
+      <Error center={true}>{t('save-credentials.unexpected-error')}</Error>
```
Comment (BigBadAlien) [inline src/components/SwapTokens/DepositAddress/DepositAddress.tsx]: Are you sure that pass `children` is ok here?

File: src/screens/Apps/AppStore/ApplicationsGridListItem/ApplicationsGridListItemStyles.ts [hunk 4]
```diff
@@ -44,6 +53,11 @@ const ApplicationsGridListItemStyles = (): StyleRules => ({
     display: 'flex',
     justifyContent: 'flex-end',
     height: 40,
+    [theme.breakpoints.down('xs')]: {
+      //  flexWrap: 'wrap',
```
Comment (BigBadAlien) [inline src/screens/Apps/AppStore/ApplicationsGridListItem/ApplicationsGridListItemStyles.ts]: //

File: src/components/SwapTokens/DepositAddress/DepositAddress.tsx (hunk 3 above)
Comment (starkyru) [reply src/components/SwapTokens/DepositAddress/DepositAddress.tsx]: There is no difference with Text component, for example

**Pull request merged**